code/modules/mob/living/simple_animal/hostile/syndicate.dm

```diff
@@ -51,7 +51,10 @@
 			var/damage = O.force
 			if(O.damtype == STAMINA)
 				damage = 0
-			health -= damage
+			if(force_threshold && damage < force_threshold)

```

Comment:
This also fixes multiple issues. The first and most obvious issue was that the "health -= damage" line simply did not work, which resulted in depot mobs being totally immune to all melee damage. Switching to adjustHealth(damage) fixes that. The second, more subtle issue was that the previous way this proc worked failed to give any notice/error when the mob was not taking damage from an attack because it was immune. These edits fix both of these problems.